tests/cli/test_doctor_tui.py

```
@@ -0,0 +1,448 @@
+            "total_gb": 100,
+            "used_gb": 50,
+            "free_gb": 50,
+            "used_pct": 50,
+        },
+    )
+    monkeypatch.setattr(
+        DOCTOR + "._load_average", lambda: {"1m": 0.1, "5m": 0.1, "15m": 0.1}
+    )
+    monkeypatch.setattr(DOCTOR + ".shutil.which", lambda name: f"/usr/bin/{name}")
+    monkeypatch.setattr(DOCTOR + ".os.path.exists", lambda p: "ssh" in p)
+    monkeypatch.setattr(DOCTOR + ".os.listdir", lambda p: ["id_a.pub", "id_b.pub"])
```

Ruff (0.16.1)

[warning] Unused lambda argument: `p`

(ARG005)